draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -198,6 +198,8 @@ Other techniques such as email or HTTP(S) based validation are out-of-scope.
 
 * `User`: the owner or operator of a domain in the DNS who needs to prove ownership of that domain to an Application Service Provider.
 
+* `DNS Adminstrator`: the owner or responsible party for the contents of a domain in the DNS.
```

Comment:
Should we express the relationship to the User?  Perhaps put this first and extend User to append ", working in coordination with their DNS Administrator"

```suggestion
* `DNS Adminstrator`: the owner or responsible party for the contents of a domain in the DNS.

* `User`: the owner or operator of a domain in the DNS who needs to prove ownership of that domain to an Application Service Provider, working in coordination with their DNS Administrator.

```

Reply:
Sure, I can see where that makes sense.  Let's add that